src/Data/Dao/PiantaDao.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+  namespace App\Data\Dao;
+  use App\Resources\Config\Database as Database;
+  use \PDO as PDO;
+  require_once __DIR__ . '/../../../resources/config/Database.php';
+
+  class PiantaDao
+  {
+
+    private $connection;
+    private $I_PLANT;
+    private $S_PLANT;
+    private $D_PLANT;
+    private $U_QR_PLANT;
+
+    public function __construct()
+    {
+      $Db = new Database();
+      $this->connection = $Db->connect();
+      $this->I_PIANTA = "INSERT INTO pianta(genere, specie, nome, co2, descrizione) VALUES(:genere, :specie, :nome, :co2, :descrizione)";
+      $this->D_PLANT = "DELETE FROM pianta WHERE id=:id";
+      $this->U_QR_PLANT = "UPDATE pianta SET qrcode = :qrcode WHERE id = :id";
+      $this->S_PLANT = "SELECT * FROM pianta AS a INNER JOIN stato_pianta AS b ON a.id = b.id_pianta WHERE a.id = :id ;";
```

Comment:
@shewraar tecnicamente all'interno di `PiantaDao` la `S_PLANT` dovrebbe restituirmi solamente il risultato della corrispondente tabella nel db e non il risultato di una `JOIN`

Reply:
Restituisco la JOIN perchè sono informazioni che riguardano la pianta e ho tutto li senza creare un altra apposita classe ecc.. così con la GET ho tutte le informazioni che riguardano quella pianta.

Reply:
eh non e cosi che funziona ... cioe quella cosa deve essere fatta in modo diverso gestendo opportunamente Entity, Model e DTO e dal metodo che fa la query deve uscire solamente un Oggetto PHP, mentre cosi restituisci un oggetto che ha all' interno delle caratteristiche della base di dati